### PR TITLE
Tests for error handling

### PR DESCRIPTION
**Pull request by 0xZenus**

### CODE REVIEW

File: tests/errorHandling.test.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
+import { expect } from 'chai';
```
Comment (ssbarbee) [inline tests/errorHandling.test.ts:1]: you are testing getReadableErrorMessage in this file. I would name the file: getReadableErrorMessage.test.ts

File: tests/errorHandling.test.ts [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+import { CRITICAL_ERROR_MESSAGE } from '../src/utils/errors/constants';
+import { getReadableErrorMessage } from '../src/utils/errors/errorHandling';
+
+describe('Error handling', () => {
```
Comment (ssbarbee) [inline tests/errorHandling.test.ts:5]:
```suggestion
describe('getReadableErrorMessage', () => {
```

File: tests/errorHandling.test.ts (hunk 1 above)
Comment (0xZenus) [reply tests/errorHandling.test.ts]: https://github.com/Voltz-Protocol/v1-sdk/pull/134/commits/5221a1df7162e7aa3a6672a6410b3c83fbab894b

File: tests/errorHandling.test.ts (hunk 2 above)
Comment (0xZenus) [reply tests/errorHandling.test.ts]: https://github.com/Voltz-Protocol/v1-sdk/pull/134/commits/5221a1df7162e7aa3a6672a6410b3c83fbab894b

**Review by ssbarbee — APPROVED** [review]

**Pull request merged**